SERVER/Server.js

```diff
@@ -242,6 +262,21 @@ server.post(Constants.LOG_TRIP_REQUEST, async function(req, res){
     logger.debug("[Server][logTrip] Coordinates    : " + JSON.stringify(data["pontos"]))
     logger.debug("[Server][logTrip] Accel data     : " + JSON.stringify(data["dados"]))
 
+
+    for(let i = 0; i < (data["pontos"]).length; i++){
+        if(data["pontos"][i][0] > 180 || data["pontos"][i][0] < -180 || data["pontos"][i][1] > 90 || data["pontos"][i][1] < -90){
+            sendErrorMessage(5, req, res);
```

Comment:
Please use error name as opposed to number, it makes reading and debugging the code easier.